pkg/option/config.go

```diff
@@ -1449,11 +1449,12 @@ type DaemonConfig struct {
 
 	// Masquerade specifies whether or not to masquerade packets from endpoints
 	// leaving the host.
-	EnableIPv4Masquerade   bool
-	EnableIPv6Masquerade   bool
-	EnableBPFMasquerade    bool
-	EnableBPFClockProbe    bool
-	EnableIPMasqAgent      bool
+	EnableIPv4Masquerade bool
+	EnableIPv6Masquerade bool
+	EnableBPFMasquerade  bool
+	EnableBPFClockProbe  bool
+	EnableIPMasqAgent    bool
+	// expired DNS lookups with still-active connections
```

Comment:
My real concern with this code change was the new comment. I understand why the other lines have changed given the new comment, but why was this comment added here? It seems unrelated to the other changes in this commit and in this PR.

Reply:
Ah, eagle's eye. Sorry, I was so careless to ignore that I added a comment by mistake. I'll just fix it.